ibis/pyspark/compiler.py

```diff
@@ -976,7 +976,8 @@ def compile_window_op(t, expr, scope, **kwargs):
 
     # If the operand is a shift op (e.g. lead, lag), Spark will set the window
     # bounds. Only set window bounds here if not a shift operation.
-    if not isinstance(operand.op(), ops.ShiftBase):
+    # Note: CumeDist doesn't work with rowsBetween here
```

Comment:
I don't understand this comment. Do you mind rephrasing this pelase?

Reply:
inside this `if`, it uses `pyspark_window.rowsBetween(start, end)` and cume_dist doesn't work with that ... not sure how to rephrase it. any idea?